js/impact.js

```diff
@@ -1334,6 +1363,14 @@ var GLPIImpact = {
                 selector       : 'node[hasITILObjects=1]',
                 onClickFunction: this.menuOnShowOngoing
             },
+            {
+                id             : 'editEdge',
+                content        : '<i class="fas fa-edit me-2"></i>' + __("Edge properties..."),
+                tooltipText    : _.unescape(__("Set name for this edge")),
```

Comment:
Special chars should be escaped, even on translations.
```suggestion
                content        : '<i class="fas fa-edit me-2"></i>' + _.escape(__("Edge properties...")),
                tooltipText    : _.escape(__("Set name for this edge")),
```